packages/console/src/pages/JwtClaims/index.tsx

```diff
@@ -43,6 +45,19 @@ function JwtClaims({ tab }: Props) {
     },
   });
 
+  const activeForm =
+    tab === JwtTokenType.UserAccessToken ? userJwtClaimsForm : machineToMachineJwtClaimsForm;
```

Comment:
nit: Consider wrapping this into a `useMemo`?

Reply:
updated